plugins/callback/foreman.py

```diff
@@ -195,13 +211,35 @@ def _ssl_verify(self, option):
 
         return verify
 
+    def _send_data(self, endpoint, host, data):
+        if endpoint == 'facts':
+            url = self.foreman_url + '/api/v2/hosts/facts'
+        elif endpoint == 'report':
+            url = self.foreman_url + '/api/v2/config_reports'
+        else:
+            self._display.warning(u'Unknown endpoint type: {type}'.format(type=endpoint))
+
+        if len(self.dir_store) > 0:
+            filename = u'{number}-{host}.json'.format(
+                    number=self.report_number, host=to_text(host))
+            filename = os.path.join(self.dir_store, filename)
+            with open(filename, 'w') as f:
+                f.write(json.dumps(data, indent=2))
+            self.report_number += 1
+        else:
+            try:
+                response = self.session.post(url=url, json=data)
+                response.raise_for_status()
+            except requests.exceptions.RequestException as err:
+                self._display.warning(u'Sending data to Foreman at {url} failed for {host}: {err}'.format(
+                    host=to_text(host), err=to_text(err), url=to_text(self.foreman_url)))
```

Comment:
Continuation line over-indented for hanging indent (E126)

I do not understand, the indentation is exactly extra one tab.

Reply:
this is not the line it's complaining about

Reply:
Thanks I just found out how to run flake8 locally, good.